### DISCUSSION TITLE
How to set up kind's configuration file

### DISCUSSION THREAD

**Discussion by Novicei** [kubernetes-sigs/kwok#695] [Q&A]
When using kind as the runtime, what flag should I use if I want to customize the configuration file of kind

**Comment by Novicei** [comment]
Does kwok support multiple nodes when running with kind? I see that when kind.yaml is formed, the number of nodes seems to be fixed

**Comment by wzshiming** [comment]
Yes, currently, that cannot be modified, what is your situation that requires this?
